addon/models/node.js

```diff
@@ -42,6 +42,7 @@ export default OsfModel.extend({
 
     root: DS.belongsTo('node', {
         inverse: null
-    })
-    //logs: DS.hasMany('node-logs'),
+    }),
+    //primaryInstitution: DS.belongsTo('institution'),
```

Comment:
Probably just part of a merge, but as a note--
Henrique has added 'affilatedInstitutions' as the 'primaryInstitution' field will be going away soon: https://github.com/CenterForOpenScience/osf.io/pull/5450